tests/testsuite/errors.rs

```diff
@@ -316,3 +316,39 @@ fn test_error_root_not_table() {
         },
     }
 }
+
+#[test]
+#[cfg(feature = "json5")]
```

Comment:
I recently made all of the tests consistently use `json` unless they are format specific.  Is there a reason this uses json5?

Reply:
done

Reply:
done